packages/@react-spectrum/dialog/docs/DialogTrigger.mdx

```diff
@@ -0,0 +1,464 @@
+    </Dialog>
+  )}
+</DialogTrigger>
+```
+
+### Dismissable
+
+If your Modal Dialog doesn't require the user to make a confirmation, you can set `isDismissable` 
+on the DialogTrigger. This adds an exit button that the user can click to dismiss the Dialog.
+
+```tsx example
+<DialogTrigger isDismissable type="modal">
+  <ActionButton>User Status</ActionButton>
+  <Dialog>
+    <Heading>Status: Bob</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        Last Login: December 12, 1989
+      </Text>
+    </Content>    
+  </Dialog>
+</DialogTrigger>
+```
+
+### Mobile type
+
+The `mobileType` prop allows you to specify what kind of Dialog should be displayed when viewed in
+mobile devices or smaller viewports. Note that Popover type Dialogs are not supported as
+a valid `mobileType`.
+
+The example below renders as a Popover in desktop but switches to a Modal in mobile. 
+
+```tsx example
+<DialogTrigger type="popover" mobileType="modal">
+  <ActionButton>Info</ActionButton>
+  <Dialog>
+    <Heading>Version Info</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        Version 1.0.0, Copyright 2020
+      </Text>
+    </Content>    
+  </Dialog>
+</DialogTrigger>
+```
+
+## Dialog Placement
+
+Popover Dialogs support a variety of placement options since they do not take over the user interface
+like Modal or Tray Dialogs.
+
+### Dialog Anchor
+
+By default, Popover Dialogs anchor themselves to their associated trigger. This can be overridden by providing
+a separate ref to the `targetRef` prop.
+
+```tsx example
+function Example() {
+  let ref = React.useRef()
+
+  return (
+    <div>
+      <DialogTrigger type="popover" targetRef={ref}>
+        <ActionButton>Trigger</ActionButton>
+        <Dialog>
+          <Heading>The Heading</Heading>
+          <Divider />
+          <Content>
+            <Text>
+              This is a popover anchored to the span.
+            </Text>
+          </Content>
+        </Dialog>
+      </DialogTrigger>
+      <span ref={ref} style={{marginInlineStart: '200px', width: '50px'}}>Popover appears over here</span>
+    </div>
+  );
+}
+```
+
+### Placement
+
+The Popover Dialog's placement with respect to its anchor element can be adjusted using the `placement`
+prop. See the props table [below](#props) for a full list of available placement combinations.
+
+```tsx example
+<DialogTrigger type="popover" placement="right top">
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover placed to the right of its trigger and offset so the arrow is at the top of the dialog.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+### Offset and Cross Offset
+
+The Popover Dialog's offset with respect to its anchor element can be adjusted using the `offset` and
+`crossOffset` props. `offset` controls the spacing between the Dialog and the anchor element parallel 
```

Comment:
It's weird to start a sentence with a lower case even when it's a lowercase word. `. 'offset' controls`